wiki/osu!tourney/osu!lazer_tournament_client/en.md

```diff
@@ -0,0 +1,87 @@
+# osu!lazer tournament client
+
+**osu!lazer tournament client** is the official client that overlays [osu!tourney](/wiki/osu!tourney) with supplementary scenes and information relevant to an osu! tournament during a live stream.
+
+Users encountering problems with the client may [create an issue on GitHub](https://github.com/ppy/osu/issues) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
+
+## Setup
+
+To start the osu!lazer tournament client, you need to specify a launch argument to the osu!lazer executable. To do this, create a new shortcut on the desktop and set the location for it to `%LOCALAPPDATA%\osulazer\osu!.exe --tournament`. This will make that specific shortcut start up osu!lazer in the tournament client mode.
+
+Because the osu!lazer tournament client is only an overlay for osu!tourney, that will need to be set up as well. Set the Release stream in osu!tourney to `Cutting Edge (Experimental)` and create an empty file called `ipc.txt` inside the installation folder of osu!tourney. After that, follow the [osu!tourney setup guide](/wiki/osu!tourney/Setup).
+
+Open the osu!lazer tournament client and you will see this setup screen:
+
+![osu!lazer tournament client setup screen](img/setup-screen.png)
+
+- Make sure the `Current IPC source` matches the location of the osu!tourney instance that you will be using.
+- Log in to osu!lazer by clicking on `Change Login`.
+- Set the correct ruleset using the dropdown menu.
+- Change the height to match the `Height` set in the `tournament.cfg` file of osu!tourney.
+
+## Managing Tournaments
+
+Tournament configurations for osu!lazer are stored in `%APPDATA%\osu\tournaments`. On first start up of the client a directory called `default` is made inside this folder. A user can maintain multiple tournament configurations and switch between them as needed to get the appropriate customisations applied.
+
+To create a new tournament configuration, create a new directory in the `tournaments` directory with the name of your tournament.
+
+Inside a tournament configuration the necessary assets can be supplied to display flags, videos and mod icons for the mappool. Each category of asset has its own folder: 
+
+- your-tournament
+    - flags
+    - mods
+    - videos
+
+## Customisation
+
+The osu!lazer tournament client can be customised by providing it with custom flags, mod icons, and video files. These will display in the respective scenes as needed.
+
+### Flags
+
+For custom flags, `.jpg` and `.png` files are accepted. These flags have to be at a resolution of 140x94 to be displayed correctly. It will be scaled to fill the shape by the client if the resolution does not match.
+
+An example of a flag using the correct specifications is this flag of Australia:
+
+![][flag_AU]
+
+Flags have to be placed in `<your-tournament>\flags`. The flags can then be referenced in the Team Editor by their filenames without the file extension.
```

Comment:
Not really sure how to handle these. I called it `your-tournament` in the example directory tree too, but I don't know if there's a way to make this more clear.